resources/views/partials/dashboard/sidebar.blade.php

```diff
@@ -63,6 +63,16 @@
                     <span>{{ trans('dashboard.metrics.metrics') }}</span>
                 </a>
             </li>
+
+            @if(subscribers_enabled())
```

Comment:
Don't disable the subscribers option here. We only disable client side because they have zero use.